melodic/distribution.yaml

```diff
@@ -13851,6 +13851,21 @@ repositories:
       url: https://github.com/locusrobotics/tf2_2d.git
       version: devel
     status: developed
+  tf2_client:
+    doc:
+      type: git
+      url: https://github.com/tpet/tf2_client.git
+      version: master
+    release:
+      tags:
+        release: release/melodic/{package}/{version}
+      url: https://gitlab.fel.cvut.cz/cras/ros-release/tf2_client.git
+      version: 0.0.0-1
```

Comment:
We don't usually like to release packages with version 0.0.0; it is pretty meaningless.  Can you please update this to have some kind of non-zero number?

Reply:
I don't have write access to that package since it's from one of my former colleagues who doesn't respond to my requests to give me write access.

I could probably change the version in the release repo via patches. Would that be okay?

Reply:
Okay, I've updated this PR and made the changes as release patches.